src/cryptoadvance/specter/templates/hwi_bridge.jinja

```diff
@@ -0,0 +1,24 @@
+{% set hwi_bridge = true %}
+{% extends "base.jinja" %}
+
+{% block sidebar %}
+{% endblock %}
+
+{% block main %}
+    <div class="card">
+        <form action="." method="POST" role="form">
+            <h1>HWI Bridge Settings</h1>
+            <label>Whitelisted domains</label>
+            <input class="form-control" placeholder="e.i. http://127.0.0.1:25441" name="whitelisted_domains" type="text" value="{{ whitelisted_domains }}">
```

Comment:
I suggest changing it to the textarea and separating domains with a new line, not with comma